parquet/src/test/java/org/apache/iceberg/parquet/TestBloomRowGroupFilter.java

```diff
@@ -184,12 +183,12 @@ public class TestBloomRowGroupFilter {
   private BlockMetaData rowGroupMetadata = null;
   private BloomFilterReader bloomStore = null;
 
-  @Rule public TemporaryFolder temp = new TemporaryFolder();
+  @TempDir private File temp;
 
-  @Before
+  @BeforeEach
   public void createInputFile() throws IOException {
-    File parquetFile = temp.newFile();
-    Assert.assertTrue(parquetFile.delete());
+    File parquetFile = temp;
```

Comment:
I think we can remove `parquetFile` here and use `temp` directly

Reply:
Removed